src/ducks/aggregations.js

```diff
@@ -0,0 +1,98 @@
+    const data = rawData.data.workflow.reductions[0];
+    dispatch({ type: FETCH_AGGREGATIONS_SUCCESS, data });
+    //--------------------------------
+
+    /*
+    const query = `{
+      workflow(id: ${workflowId}) {
+        reductions(subjectId: ${subjectId}, reducerKey:"${reducerKey}") {
```

Comment:
The end-point is `subject_reductions` on staging (this is because `user_reductions` will be a thing soon on Caesar).